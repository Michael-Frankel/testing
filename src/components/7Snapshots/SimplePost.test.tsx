import { SimplePost } from './SimplePost'
import { render } from '@testing-library/react'

describe('Simple post snapshot tests', () => {
    it('initial test', () => {
        const rendered = render(
            <SimplePost
                content='Simple content'
                user='Alexei'
            />
        );
        expect(rendered.asFragment()).toMatchSnapshot();
    })
})