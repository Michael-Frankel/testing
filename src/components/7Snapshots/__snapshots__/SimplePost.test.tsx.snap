// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Simple post snapshot tests initial test 1`] = `
<DocumentFragment>
  <div
    data-testid="post-container"
  >
    <h2>
      Alexei:
    </h2>
    <p>
      Simple content
    </p>
  </div>
</DocumentFragment>
`;
